packages/fxa-auth-server/lib/subhub/client.js

```diff
@@ -279,5 +299,19 @@ module.exports = function(log, config) {
         throw err;
       }
     },
+
+    async deleteCustomer(uid) {
+      try {
+        return await api.deleteCustomer(uid);
+      } catch (err) {
+        log.error('subhub.deleteCustomer.1', { uid, err });
```

Comment:
`.1`? Just wondering if this is another pattern I'm not familiar with.

Reply:
I was wondering the same thing TBH

Reply:
I confess, I was just following the convention from the rest of the module here, although I've seen it elsewhere in the auth server too. Typically when writing fresh modules from scratch, I prefer to use more descriptive error names. I'll do that here too instead, thanks for the nudge.